src/main/java/sk/freemap/gpxAnimator/Option.java

```diff
@@ -38,9 +39,10 @@
 	SKIP_IDLE("skip-idle", "idle-skipping flashback effect duration in milliseconds; set to empty for no flashback"),
 	PHOTOS("photos", "a directory containing photos to be added to the animation (must contain EXIF information with date and time of photo taken)"),
 	PHOTO_TIME("photo-time", "the amount of time, a photo should be shown above the map"),
-	HELP("help", "this help");
-	
-	private static java.util.Map<String, Option> map = new HashMap<String, Option>();
+	HELP("help", "this help"),
+	TAIL_COLOR("tail-color", "highlighted tail color");
+
+    private static java.util.Map<String, Option> map = new HashMap<String, Option>();
```

Comment:
Mixing of code formatting changes with code changes is hard to review and to spot changed code.